shared/core/src/listings.ts

```diff
@@ -15,6 +16,7 @@ export interface Attachment {
 export interface Listing {
   acceptingApplicationsAtLeasingAgent: boolean
   acceptingApplicationsByPoBox: boolean
+  acceptingOnlineApplications: boolean
```

Comment:
do we need to maintain this as a separate boolean, or can we just derive it from the existence of an Application URL?

@slowbot - food for future thought: what would it mean for the UI in different combinations of application methods here? Could we have a listing that people could apply to in multiple ways (e.g. online or paper)? How should we render in that case? (note: we do not need to answer these questions now to merge, just want to surface them while the iron is hot)

Reply:
re: the first question… I feel like some of these booleans could be set by a smarter backend rather than being data we're implicitly storing. Not sure how we want to handle it at present.